Update

gustavosr8 opened the pull request.

ericonr commented on iocBoot/iocIpmiMgr/stIpmiMgr.cmd:
```diff
@@ -13,7 +13,6 @@ drvAsynIPPortConfigure ("$(PORT)","$(IPADDR):623 udp",0,0,0)
 mchInit("$(PORT)")
 
 ## Load record instances
-dbLoadRecords("$(ASYN)/db/asynRecord.db","P=${P}, R=${R}asyn,PORT=$(PORT),ADDR=0,OMAX=80,IMAX=80")
```
The commit with this removal is wrong.

Since the IOC uses `drvAsynIPPort`, it depends directly on Asyn. And the asyn record is useful to track information like connection (status of `$(P)$(R)asyn.CNCT`), and allows debugging as well.

ericonr commented on ipmiMgrApp/Db/afcv4.substitutions:
```diff
@@ -0,0 +1,44 @@
+#            { FMC1Curr3V3     , 3           , 3       , $(fruid)     , $(id)  }
+#            { FMC2Curr12V     , 4           , 3       , $(fruid)     , $(id)  }
+#            { FMC2CurrVADJ    , 5           , 3       , $(fruid)     , $(id)  }
+#            { FMC2Curr3V3     , 6           , 3       , $(fruid)     , $(id)  }
```
I would remove these entirely. The commit message should mention that we should be able to revert this commit eventually.

ericonr commented on ipmiMgrApp/Db/microtca_bpm_crate.substitutions:
```diff
@@ -58,6 +58,12 @@ file "afc.db"
+
 {
     pattern { id       , fruid }
             { RtmTim   , 90    }
```
This diff is broken, it's pushing the `rtm_timing.db` line away from its pattern.

ericonr commented on ipmiMgrApp/Db/mch_nat.substitutions:
```diff
@@ -27,23 +27,23 @@
 {
     pattern { id           , fruid }
-            { $(id)Base    , 3     }
+            { Base    , 3     }
```
Please explain why this is wrong in the commit message.

henriquesimoes replied:
I feel like the message could be a little bit more direct to the point. Not a specific suggestion, though.

henriquesimoes replied:
Have you taken another look at this?

At least a typo should be resolved: `s/them/then/`.

ericonr reviewed (changes requested):
Some of the commits have the wrong spelling for "separate" (though I'd use "split").

gustavosr8 committed:
Change the default sensors scan period to 5 seconds

Since we want to have the sensors being monitored all the time, we
should use the maximum acquire frequency to have as much information as
possible.

gustavosr8 committed:
Split RTM board support into LAMP and 8SFP

Since now we have two different RTM devices, we can should separate the
boards in two different .substitutions files.

gustavosr8 committed:
Split afcs into v3 and v4

Since afcv3 and afcv4 have different sensor readings, a different file
for each version is needed. This commit may be reverted in the future,
when support for new sensors is added, specially voltage and current in
afcv4.

gustavosr8 committed:
Remove rescan record

The rescan command isn't supported by the IPMIComm driver.

gustavosr8 committed:
Change the $(id) field to $(suffix)

Expanding $(id) to $(id)Suffix was leading to a duplicated Suffix in the
resulting db. So, using another variable name to expand this fields
solves this.

gustavosr8 committed:
Remove unsupported records

gustavosr8 committed:
Rename the power modules (PM) and the MCH to match the ones in MCH shell

henriquesimoes commented on ipmiMgrApp/Db/system_common.template:
```diff
@@ -244,8 +244,8 @@ record(mbbo, "$(P)$(R)$(id)SensorScanPeriod-Sel") {
- field(VAL,  "1")
- field(RVAL, "1")
+ field(VAL,  "0")
+ field(RVAL, "0")
```
There is a typo in the commit message: `s/beeing/being/`.

henriquesimoes commented on ipmiMgrApp/Db/Makefile:
```diff
@@ -28,7 +28,8 @@ DB += mch_clk_switch.db
 DB += shelf.db
-DB += afc.db
+DB += afcv3.db
+DB += afcv4.db
```
Typo in `differente` in the commit message.

henriquesimoes reviewed (commented):
Skimming through the PR I found some typos in the commit messages.

ericonr commented on ipmiMgrApp/Db/system_common.template:
```diff
@@ -244,8 +244,8 @@ record(mbbo, "$(P)$(R)$(id)SensorScanPeriod-Sel") {
- field(VAL,  "1")
- field(RVAL, "1")
+ field(VAL,  "0")
+ field(RVAL, "0")
```
Ok, I think we should only merge this change if we can confirm that adding enough delay between messages to avoid MCH errors still allows it to run in less than 5s.

gustavosr8 replied:
We found out that the MCH warnings is not being caused because there are a lot of requisitions nor the delay between this requisitions. The main hypothesis is that it is from some statistical misleading communication between MCH and MMC, so adding delay between messages don't fix the problem

ericonr reviewed (commented):
Commit message nits:

"we can better separate" -> "we should separate"

"a separated supportn to each version is neaded" -> "a different file for each version is needed"
"may be undone" -> "may be reverted"
"support to new" -> "support for new"

"remove unused records" -> "remove unsupported records" ? that seems more accurate
